### PR TITLE
New branches

### PR DESCRIPTION
**Pull request by yiiyama**
See panda.def for the changes.

### CODE REVIEW

File: defs/panda.def [hunk 1]
```diff
@@ -237,6 +237,22 @@ enum TriggerObject {
...
   fPh36EBR9Iso
 };
+enum Selector {
+  kCutBasedIdVeto,
+  kCutBasedIdLoose,
+  kCutBasedIdMedium,
+  kCutBasedIdTight,
+  kCutBasedIdHLTSafe,
+  kMvaWP90,
+  kMvaWP80,
```
Comment (dabercro) [inline defs/panda.def]: This there a difference between how these are filled, and these, for example? https://github.com/yiiyama/PandaTree/blob/39df5025b4e422d97b0b38970b76ed5348e716d8/defs/panda.def#L279-L280

Comment (yiiyama) [reply defs/panda.def]: No, the pre-existing ID bit branches are kept for backward compatibility. If we are happy with the boolean arrays, we should drop the old branches in the next version.

Comment (yiiyama) [reply defs/panda.def]: Sorry, wasn't clear. There is no difference in the way the old and new booleans are filled (in fact one copies the other's value in XYZFiller.cc).

Comment (dabercro) [reply defs/panda.def]: Why do we want to switch to arrays? I think this is still susceptible to my complaint in the email about shifting values in the headers. In addition, we can't quickly check plots like this: http://t3serv001.mit.edu/~jenkins/relval/PandaPhysics/PandaTree/master/3/SingleMuon/muons/loose.pdf and the output of `dump()` (which I've fallen into the habit of looking at a lot) will be a lot less useful if we ever make a full switch.

Comment (yiiyama) [reply defs/panda.def]: Yeah I see your point and I'm OK with making one branch per flag too. The array makes the list of branches more manageable and in some cases make the code a little more compact (if you need to combine them) but indeed the downside is that the indices can shift, and if we add elements to the enum in the next versions, that breaks the backward compatibility.

**Comment by mitsidekick** [comment]
Tests passed! Plots here:

- http://t3serv001.mit.edu/~jenkins/relval/?d=PandaPhysics/PandaTree/PR-115/1

**Comment by mitsidekick** [comment]
Tests passed! Plots here:

- http://t3serv001.mit.edu/~jenkins/relval/?d=PandaPhysics/PandaTree/PR-115/2

**Comment by yiiyama** [comment]
I think I'm done adding new branches. Please merge if you guys are OK with the new list of variables.

**Comment by mitsidekick** [comment]
Tests passed! Plots here:

- http://t3serv001.mit.edu/~jenkins/relval/?d=PandaPhysics/PandaTree/PR-115/3

**Comment by dabercro** [comment]
I'm happy. If @DylanHsu or @sidnarayanan don't say otherwise tonight, let's merge it tomorrow morning.

**Review by sidnarayanan — APPROVED** [review]
Yup, this is fine by me.

**Review by dabercro — APPROVED** [review]

**Pull request merged**